src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball randomBall = new Ball();
+        randomBall.setColor(new ColorSupplier().getRandomColor());
+        randomBall.setNumber(new Random().nextInt(100));
```

Comment:
Random object creates every time you call getRandomColor. Think of the approach, where you create an object of Random only once for this class
ColorSupplier object creates every time you call getRandomFigure. Think of approach, when you create an object of ColorSupplier only once for this class